app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,15 @@
+class LikesController < ApplicationController
+  def create
+    @post = Post.find(params[:post_id])
+    new_like = current_user.likes.new(
+      user_id: current_user.id,
+      post_id: @post.id
+    )
+    new_like.update_likes_counter
+    if new_like.save
+      redirect_to "/users/#{@post.user_id}/posts/#{@post.id}", notice: 'Success!'
+    else
+      render :new, alert: 'Error occured!'
+    end
+  end
```

Comment:
- [ ] Kindly, fix the create method for the likes, when a user clicks on the like button the likes are being counted however it is causing this issue, please fix it.

![image](https://user-images.githubusercontent.com/55635977/154975387-737fc5b6-2cbf-41a8-841f-e537f1635619.png)

Reply:
Thanks sir, its fixed